platforms/documentation/docs/src/docs/userguide/authoring-builds/gradle-properties/service_injection.adoc

```diff
@@ -35,6 +35,7 @@ The following services are available for injection:
 8. <<execoperations,`ExecOperations`>> - Allows a task to run external processes with dedicated support for running external `java` programs.
 9. <<toolingmodelbuilderregistry,`ToolingModelBuilderRegistry`>> - Allows a plugin to registers a Gradle tooling API model.
 10. <<testeventreporterfactory,`TestEventReporterFactory`>> - Allows a plugin to access Gradle's test events and its corresponding API.
+11. <<dependencyfactory,`DependencyFactory`>> - Allows a plugin to create dependencies in a type-safe way.
```

Comment:
This list has become confusing with the exceptions listed below.

I think technically, `DependencyFactory` and `TestEventReporterFactory` and maybe some others aren't available in workers.

Reply:
Maybe something like this:
https://github.com/gradle/gradle/pull/33954/files

cc @lkasso